Lib/test/test_ast/test_ast.py

```diff
@@ -1614,6 +1614,12 @@ def test_literal_eval_syntax_errors(self):
                 (\
             \ ''')
 
+    def test_literal_eval_large_input_crash(self):
+        # gh-125010: Fix use-after-free in ast repr()
+        source = "{0x0" + "e" * 250_000 + "%" + "e" * 250_000 + "1j}"
```

Comment:
Should we actually hide this behind a bigmem resource?

Reply:
Creating and eval'ing this string takes less than a second and consumes about 1 MB of memory. Therefore, I think we do not need to mark this test case as requiring any resources.